databox/api/tests/MetadataManipulator/ReaderTest.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Tests\MetadataManipulator;
+
+use Alchemy\ApiTest\ApiTestCase;
+use Alchemy\MetadataManipulatorBundle\MetadataManipulator;
+use PHPExiftool\Driver\Metadata\Metadata;
+use PHPExiftool\Driver\Metadata\MetadataBag;
+use PHPExiftool\Driver\TagGroup\IFD0\Artist;
+use PHPExiftool\Reader;
+
+class ReaderTest extends ApiTestCase
```

Comment:
This test should be in the bundle instead.

Reply:
ok